tests/model_serving/model_server/kserve/negative/test_concurrent_invalid_requests.py

```
@@ -0,0 +1,182 @@
+"""
+Tests for system stability under concurrent invalid inference requests.
+
+Boundary condition: sending a burst of malformed/invalid requests concurrently
+stresses the server's error-handling path without any valid traffic.  The goals
+are:
+
+1. Every concurrent request receives a 4xx (client-error) response – the server
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)